### DISCUSSION TITLE
暑假做什麼呢？

### DISCUSSION THREAD

**Discussion by JessyCheng2003** [1101-GS4719-C/Python-Community-GS4719-C#1422] [General]
每天都在期待暑假的我，一直在想暑假可以做什麼～這邊提供幾個建議，讓大家充實暑假！
1.旅行：體驗不同國家、不同文化，開闊眼界。
2.做志工：積極回饋社會。
3.閱讀：增進內涵和知識。
4.健身：在學期間都很忙碌，沒有時間好好運動，大家可以趁暑假多多運動，保持健康！
5.暑修：把之前被當的課補回來。
6.實習或打工：賺錢以外還可以增加社會經驗。
7.創意：寫文章、畫畫、攝影等等，激發自己的創造力。
8.自學：趁暑假的時候，可以去學一些外語或是培養新的技能。

以上是我想到的一些適合暑假做的事～那大家暑假有想要或規劃做什麼嗎？

**Comment by Superlan8** [comment]
今年暑假我想當個全職考生，專心備戰GRE，暑假我認為是能分辨大學生成功與否的關鍵，有認真規畫暑假的人跟擺爛的人會差得非常多，畢竟大學暑假有接近三個月，這個時間對於學習或是累積其他經驗是非常足夠的。

**Comment by ALAN30359704** [comment]
暑假對我來說是個自我學習的時間，是個能夠找到自己真正有興趣或專長的好機會，因為平時上課常常是為了分數或者是老師或家人的要求，但是暑假反而無關與分數或者是家人的逼迫可以讓我們自由的規劃去完成想做的事！

**Comment by zephyrxin01** [comment]
暑假的確有很大的空間可以讓自己安排想做的事情，但也很容易一不小心虛度時光。我目前除了想好好複習重要的科目外，也希望自己可以養成運動的習慣，健康還是很重要的！

**Comment by youtinglin** [comment]
身為大學生的我們，不得不說暑假真的是蠻長的，是可以好好利用以充實自己的時光，而我本身會去業界實習，賺點錢的同時也能累積硬實力，還能增加自己未來的競爭力，於此同時也是聯絡家人感情的時機，可以規劃家庭旅遊，且因疫情減緩可出國看看，擴展自己的視野，更棒的是可以做一些平常在學期間太忙無法做的事，把生活重心放到自己的興趣愛好上。

**Comment by Elaine2899** [comment]
這是我在中央的第一個暑假，在學期中我明顯感受到自己的不足，包含語言能力、軟實力以及軟體能力。因此我預期在這第一個暑期中可以從頭自學英文，以補足過往在英文學習上的缺失，也期許自己可以接一些較簡單的家教來增加自己的自信心、口條能力以及組織能力。最後還希望可以在軟體能力方面也能有所增強，比如PYTHON希望能增加數據分析方面的能力。

**Comment by brian431** [comment]
首先旅行是一定要的，暑假通常會安排一次跟家人出國一次跟朋友國內旅遊。再來就是好好彌補學期中沒時間看的劇、漫畫、電影及沒時間玩的遊戲。最後則是因為罪惡感而稍微學習。

**Comment by Aaron0810** [comment]
大家怎麼都這麼認真，我大三之後應該也是就在學校準備明年考研就是了，但前兩年暑假我幾乎都沒甚麼在念書，也特別做甚麼特別有意義的事，大部分都在玩就是了，我相信樓上的各位雖然都說得頭頭是道，會不會做應該是另外一回事，但玩是肯定會玩的吧。

**Comment by juliaouo** [comment]
這個暑假我考慮去當實習生，累積一點實務經驗，畢竟快畢業了，為前幾年的暑假都沒有好好利用起來感到有點可惜，但其實也不會太後悔，畢竟生涯中也不剩幾個暑假了，不論是玩也好，學習新事物也好，重要的是不要愧對自己，不要後悔。

**Comment by 111607004** [comment]
其實我覺得拿暑假來跑營隊也是個不錯的選擇，可以在營隊裡面認識不同的人，結交到志同道合的朋友，而且我很喜歡那種和大家一起完成一件事情的感覺，跑完後雖然很累，但留下的是滿滿的回憶還有無法計數的成就感。

**Comment by RogelioKG** [comment]
@Aaron0810 大實話。但想和大家分享我的故事。
記得前年與大前年的暑假，因為重考壓力很大，我就是在家耍廢打魔獸。
直到去年 4 月。
我赫然發覺與我同屆的朋友已經大二了，
而我，思維和知識全部停在高三，彷彿我周遭的時間暫停了。
不知道那時候是想通了？又或者是腦袋不對勁？
放在我電腦桌上的是一本尚未翻開的《精通 Python》，
我永遠記得那天，是 4 月 1 號。
30 天後，我幫我朋友完成他資工系教授出的 Python 作業，題目是用 class 做一個德州撲克。
5 月初時，我決定開始自學 C，即便這些概念在我眼中像是天書一般。
桌上擺的是另一本《C Primer Plus》，有念過的朋友應該知道這是一本 800 多頁的磚頭書。
在做完標準庫函式的筆記與 stack / queue / tree 的手刻練習後，闔上了最後一頁。
那個時候是 7 月初。從我入門開始僅經過 90 天。
我入門一段時間後就察覺我並不是那麼有天賦，簡單來說就是：我很笨 (不然我也不會重考)。
我朋友所謂的「彎道超車」，是我在每個睡不著的夜晚敲著鍵盤的所努力的結果。
早上起床張開眼睛就是程式，刷牙也在想程式，作夢也在解程式，
你朋友在打遊戲，你要打程式。你朋友跑出去騷兩圈，你要打程式。
就連和朋友出去打麻將，也是想著能不能讓程式幫我判斷我這樣贏幾台 (並沒有)。

暑假有很多時間，有不少朋友也想要自學，但卻發現自己沒什麼動力或者效率。
希望小弟的經驗能給你們一些建議：
`找到你的目標，那會是你的動力`
最好能與那些能夠持續交流的朋友共同成長。
接著你必須對他信心喊話。例如：「我 60 天就會超越你，你信不信」；
或者有個唱衰你的朋友，他們越覺得你不可能辦到，越是要做給他們看。
我沒有鼓吹什麼活在他人的眼光裡還是心靈雞湯之類的。
渴望自我成長卻缺乏動力，是不可能達成目標的。
嘗試努力證明自己，就是最好的激將法。
供參。

**Comment by xiang305** [comment]
我這個暑假會有一部分的時間留在學校跑營隊，雖然籌備過程真的很累，可能連續4天只能睡大概5個小時，但嘗試新鮮事物的感覺真的很棒，還能多認識幾個朋友。另外我也參加了圍棋棋院的師培訓練，想說發揮一下我為數不多的專長，也拓寬一下自己未來的選擇
